academic_helper/models/course_occurrence.py

```diff
@@ -92,10 +92,20 @@ class ClassGroup(Base):
     occurrence = models.ForeignKey(CourseOccurrence, on_delete=models.CASCADE)
     class_type = models.IntegerField(choices=ClassType.list())
     mark = models.CharField(max_length=30, null=True, blank=True)
+    teachers = models.ManyToManyField(Teacher, blank=True)
 
     def __str__(self):
         return f"{self.occurrence} - {ClassType(self.class_type).readable_name} ({self.mark})"
 
+    class Meta:
+        unique_together = ["occurrence", "class_type", "mark"]
```

Comment:
Looks like you forgot the migration for that. You can just delete migration `0011` and run again `makemigrations`. Your database will then wont work, just delete it and run `migrate`.